puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/katello-devel-forklift-test.yaml

```diff
@@ -0,0 +1,14 @@
+
+- job:
+    name: katello-devel-forklift-test
+    project-type: pipeline
+    triggers:
+      - reverse:
+          jobs:
+            - foreman-nightly-rpm-pipeline
```

Comment:
I think you want the `katello-nightly-rpm-pipeline` to ensure you are testing against latest Katello bits?